### ISSUE TITLE
Reverse Proxy config not working

### ISSUE THREAD

**Issue by seanvree** [scytherswings/Plex-Board#86]
When setting the releative_root in server_config for reverse proxy, one of three failures will happen:

1:
-----------------------------
relative_root: /plexstatus
web_host: http://FQDN,com:3000
asset_host: http://FQDN.com:3000

The application will not launch, and the prodcution.log will be filled with these lines continuously:

```
I, [2017-04-12T10:45:28.515768 #22288]  INFO -- : Setting asset_host to: http://FQDN.com:3000
I, [2017-04-12T10:45:28.515768 #22288]  INFO -- : Setting relative_root to: /plexstatus
```

2:
----------

If you set to the server_Config to an internal IP : PORT with relavtive_Root, everything works on the server, but you might see these errors in a client broswer:

HTTP Error 502.3 - Bad Gateway
A security error occurred

FQDN.com/ Failed to load resource: the server responded with a status of 502 (Bad Gateway)

3:
-------

However, if you set it to NO relative path and an internal IP and port - the application frontpage will locad, but  you might get these error:

```
I, [2017-04-12T10:14:39.756266 #26700]  INFO -- : Started GET "/assets/application-9db612323a699b3d704a95668edafd322def5db9a380d15959f3c8f7e799f11d.css" for 192.168.1.20 at 2017-04-12 10:14:39 -0700
I, [2017-04-12T10:14:39.758229 #26700]  INFO -- : Started GET "/assets/application-f8aba3e29220e9042f2c06c1d15b37240e7adb57703c8d2a92e19244d0f405f4.js" for 192.168.1.20 at 2017-04-12 10:14:39 -0700
I, [2017-04-12T10:14:40.062441 #26700]  INFO -- : Started GET "/assets/bootstrap/glyphicons-halflings-regular-fe185d11a49676890d47bb783312a0cda5a44c4039214094e7957b4c040ef11c.woff2" for 192.168.1.20 at 2017-04-12 10:14:40 -0700
I, [2017-04-12T10:14:40.172352 #26700]  INFO -- : Started GET "/plexstatus/notifications" for 192.168.1.20 at 2017-04-12 10:14:40 -0700
F, [2017-04-12T10:14:40.173333 #26700] FATAL -- : 
ActionController::RoutingError (No route matches [GET] "/plexstatus/notifications"):
  actionpack (4.2.8) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'
  actionpack (4.2.8) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  c:/RailsInstaller/Ruby2.3.0/lib/ruby/gems/2.3.0/bundler/gems/browserlog-7b4eabc2649b/lib/browserlog/selective_logger.rb:48:in `default_behaviour'
  c:/RailsInstaller/Ruby2.3.0/lib/ruby/gems/2.3.0/bundler/gems/browserlog-7b4eabc2649b/lib/browserlog/selective_logger.rb:28:in `call_app'
  railties (4.2.8) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.2.8) lib/active_support/tagged_logging.rb:68:in `block in tagged'
  activesupport (4.2.8) lib/active_support/tagged_logging.rb:26:in `tagged'
  activesupport (4.2.8) lib/active_support/tagged_logging.rb:68:in `tagged'
  railties (4.2.8) lib/rails/rack/logger.rb:20:in `call'
  actionpack (4.2.8) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.6.5) lib/rack/methodoverride.rb:22:in `call'
  rack (1.6.5) lib/rack/runtime.rb:18:in `call'
  actionpack (4.2.8) lib/action_dispatch/middleware/static.rb:120:in `call'
  rack (1.6.5) lib/rack/sendfile.rb:113:in `call'
  railties (4.2.8) lib/rails/engine.rb:518:in `call'
  railties (4.2.8) lib/rails/application.rb:165:in `call'
  rack (1.6.5) lib/rack/urlmap.rb:66:in `block in call'
  rack (1.6.5) lib/rack/urlmap.rb:50:in `each'
  rack (1.6.5) lib/rack/urlmap.rb:50:in `call'
  puma (3.8.2) lib/puma/configuration.rb:224:in `call'
  puma (3.8.2) lib/puma/server.rb:600:in `handle_request'
  puma (3.8.2) lib/puma/server.rb:435:in `process_client'
  puma (3.8.2) lib/puma/server.rb:299:in `block in run'
  puma (3.8.2) lib/puma/thread_pool.rb:120:in `block in spawn_thread'

````

**Comment by scytherswings** [comment]
This should be fixed in v0.12.0 on `dev`. I'm terrible at tagging commit messages. Let me know if those changes don't work. I tested them with `nginx` on my Ubuntu box so I thought they worked ok.

**Comment by scytherswings** [comment]
Tested reverse proxy functionality with nginx along with relative_root and v0.12.0 works again.
